### DISCUSSION TITLE
gmake/gmake2

### DISCUSSION THREAD

**Discussion by Jarod42** [premake/premake-core#1693] [Q&A] [answered]
Which is the rationale to keep both?

- Does gmake support something that gmake2 doesn't?
- should bug-fix should be applied to gmake?
- Can we remove deprecated one? (we can still have alias if you want to keep both names for retro-compatibility)

(testing custom rule, gmake2 supports them whereas gmake doesn't).

**Comment by samsinsane** [comment, marked as answer]
#1099.
